Src/Plugins/ValidatorBase.cs

```diff
@@ -181,6 +189,30 @@ public static string ReturnUnknownHost(ref string message, string host)
             return null;
         }
 
+        private static string ParseValue(string value)
+        {
+            if (string.IsNullOrEmpty(value))
+            {
+                return value;
+            }
+
+            value = value.Substring(value.IndexOf('=') + 1);
+
+            int index = 0;
+            foreach (char ch in value)
+            {
+                if (ch == ' ' || ch == '\t')
+                {
+                    index++;
+                    continue;
+                }
+
+                break;
+            }
+
+            return value.Substring(index);
```

Comment:
>Substring [](start = 25, length = 9)

should this helper call trim() or otherwise ensure no trailing white space? Also, shouldn't this helper be internal and unit-tested? easy to do.